core/src/html/text_format.rs

```diff
@@ -717,7 +717,34 @@ impl FormatSpans {
                             }
 
                             if let Some(size) = attribute(b"size") {
-                                format.size = size.parse().ok();
+                                // if the number starts with + or -, the size is relative
+                                let (prefix, size) = if size.starts_with(&[b'+', b'-'][..]) {
+                                    (size.at(0) as u8, &size[1..])
+                                } else {
+                                    (0u8, &size[..])
```

Comment:
Let's use Some(char) and None.

Reply:
Is `Option<u8>` okay? For char I would have to decode the text or do `as u8 as char`, but we know here that the first character is ASCII for sure

Reply:
Sounds good. `Option<u16>` would be an option, but that seems worse for matching later.